File: Pemrograman 1/MODUL 3/Bahasa Python/PRAK304-2410817110002-Akhmad_Daffa_Azzikri.py
a = int(input(""))

if a == 0:
    print("Nol")
elif 1 <= a <= 9:
    print("satuan")
elif a == 10 or 20 <= a <= 99:
    print("puluhan")
elif 11 <= a <= 19:
    print("belasan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Pemrograman 1/MODUL 3/Bahasa C/PRAK304-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

int main(){
	int a;
	
	printf("");
	scanf("%d",&a);
	
	if (a == 0){
		printf("Nol");
	}
	else if (a>= 1 && a<=9){
		printf("satuan");
	}
	else if (a == 10 || a>= 20 && a<= 99){
		printf("puluhan");
	}
	else if (a >= 11 && a<= 19){
		printf("belasan");
	}
	else if(a >= 100){
		printf("Anda Menginput Melebihi Limit Bilangan");
	}
	
	return 0;
}
		